internal/oscal/generate_test.go

```diff
@@ -1,10 +1,41 @@
 package oscal
 
 import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+	"strings"
 	"testing"
 )
 
-// TestFmtFieldName tests that we handle formatting a json string to a go struct correctly
+// TestOscalSchemaVersion tests that the OSCAL schema version is correct.
+func TestOscalSchemaVersion(t *testing.T) {
+	oscalSchemaVersion := "1.0.4"
```

Comment:
No issues here - but would like to keep this on the radar for helping determine what a strong test looks like in the future - can we test multiple versions of OSCAL using this logic? I would assume so?

Reply:
This logic wouldn't allow to test against multiple versions as is, but I don't think it would be too difficult to refactor this test to allow for checking multiple versions. I would think we could just read the string from each schema file, store them to a string slice, loop through them, and check `if` they contain the proper version. Maybe I'm misunderstanding how that would work though?

Reply:
My assumption is that a different version would mean a different file entirely

Reply:
definitely! my intent for the comment was thinking more about what the workflow would look like (from the user perspective) and how we might track supported versions of OSCAL in the source code - No changes to make right now though.